modules/controlfreec/1.2/etc/scripts/freec2circos.pl

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env perl -w
```

Comment:
This looks handy, but I couldn't find a rule in the Snakefile to generate circos plots?

Reply:
I haven't implemented this part yet. I can throw it in though if you want that as an option. It just translates the ratio file into a Circos track

Reply:
(It doesn't actually generate the circos plot though)